src/components/Filter.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { STATUSES, AllProductRequestProps, ProductRequest } from 'types';
+import '../index.scss';
+
+interface IProps {
+  productRequests: ProductRequest[];
+  showData: ProductRequest[];
+  setShowData: any;
+}
+
+export const Filter = (props: IProps) => {
+  const { productRequests } = props;
+  const { showData } = props;
+  const { setShowData } = props;
+
+  const showAll = () => {
```

Comment:
What would be the return type for this function?

Reply:
Void because I'm not returning anything.